ui/components/multichain/avatar-group/avatar-group.tsx

```diff
@@ -35,6 +35,7 @@ export const AvatarGroup: React.FC<AvatarGroupProps> = ({
     marginLeftValue = '-16px';
   }
   const tagValue = `+${(membersCount - limit).toLocaleString()}`;
+  console.log(tagValue, typeof tagValue, showTag);
```

Comment:
This is removed here https://github.com/MetaMask/metamask-extension/pull/21342/commits/454b49938fdc7bdafe0f30f255604ba81c59d4c3